packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -51,7 +51,9 @@ const headingTransformations = [
   { from: "fontWeightHeadingTitle6", to: "headingTitle6FontWeight" },
   // keep type and tailwind classes to avoid visual breaking changes
   { from: 'type="title1"', to: 'type="title0"' },
+  { from: 'type: "title1"', to: 'type: "title0"' },
   { from: "leading-heading-title1", to: "leading-heading-title0" },
+  { from: "theme(lineHeight.heading-title1)", to: "theme(lineHeight.heading-title0)" },
```

Comment:
This sure covers Vojta's note:
```
theme(lineHeight.heading-title1) doesn’t get caught by the codemod.
```

but I am not sure if the change is 100% correct (based on my understanding). 🙈  Please, check it out. 🙏

Reply:
Yes, that is what is expected